test/e2e/tests/test_namespace_scoping.py

```
@@ -114,6 +120,13 @@ def _create_namespace(name: str):
     if result.returncode != 0 and "already exists" not in result.stderr:
         raise RuntimeError(f"Failed to create namespace {name}: {result.stderr}")
 
+    if tenant_enabled:
+        subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)